src/main/java/uk/gov/hmcts/reform/sscs/helper/SscsDataHelper.java

```diff
@@ -64,7 +67,11 @@ public void addSscsDataToMap(Map<String, Object> appealData, Appeal appeal, List
                         appeal.getBenefitType().getCode(),
                         appeal.getMrnDetails().getDwpIssuingOffice());
 
-                    officeMapping.ifPresent(office -> appealData.put("dwpRegionalCentre", office.getMapping().getDwpRegionCentre()));
+                    String dwpRegionCentre = "PIP".equalsIgnoreCase(appeal.getBenefitType().getCode())
```

Comment:
Same logic is used in tribunal-api, I think its better to add this common logic to sscs-common

Reply:
Done